.github/workflows/test.yml

```diff
@@ -13,138 +13,36 @@ on:
       - release-**
 
 jobs:
-  prepare_dependencies:
-    name: Prepare dependencies
+  test_repository:
+    name: Run linter, build and test all packages
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
 
       - name: Setup node
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: '12'
-
-      - name: Get cached dependencies
-        uses: actions/cache@v2
-        with:
-          path: '**/node_modules'
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          cache: 'yarn'
 
       - name: Install dependencies
-        run: yarn --network-concurrency 1 --frozen-lockfile
-
-  lint:
-    name: Lint
-    needs: prepare_dependencies
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Setup node
-        uses: actions/setup-node@v1
-        with:
-          node-version: '12'
-
-      - name: Get cached dependencies
-        uses: actions/cache@v2
-        with:
-          path: '**/node_modules'
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        run: yarn install --frozen-lockfile
 
       - name: Run linter
         run: yarn lint
-
-  validate_core:
-    name: Validate core
-    needs: prepare_dependencies
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-
-      - name: Setup node
-        uses: actions/setup-node@v1
-        with:
-          node-version: '12'
-
-      - name: Get cached dependencies
-        uses: actions/cache@v2
-        with:
-          path: '**/node_modules'
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-
+      
       - name: Build core
         run: yarn build:core
 
+      - name: Test core
+        run: yarn test:core --coverage
+
       - name: Test cache
         run: yarn test:cache --coverage
 
       - name: Test CLI
         run: yarn test:cli --coverage
 
-      - name: Test core
-        run: yarn test:core --coverage
-
       - name: Test middleware
         run: yarn test:middleware --coverage
-
-  validate_integrations:
```

Comment:
What is the reason for removing it?

Reply:
We no longer maintain boilerplate, and commercetools on this repo, and both will be deleted soon. There is no point in running tests for them here.

Reply:
Ok, that's what i've thought